README.md

```diff
@@ -199,7 +199,7 @@ for the user, relying on reasonable defaults in many cases.
 The RubySMB Client is capabale of multi-protocol negotiation. The user simply specifies whether SMB1 and/or SMB2 should be supported, and
 the client will negotiate the propper protocol and dialect behind the scenes.
 
-In the below example, we tell the client that both SMb1 and SMB2 should be supported. The Client will then Negotiate with the
+In the below example, we tell the client that both SMB1 and SMB2 should be supported. The Client will then Negotiate with the
```

Comment:
Why are `Client` and `Negotiate` capitalized? They aren't in the previous sentence.

Reply:
This whole doc could use a scrub in hindsight. This might be because it was referring to the class RubySMB::Client rather than the general concept of a client. That, or someone wrote this in an IDE and it autocorrected it as capitalized. My money is on the second theory.